app/controllers/friendships_controller.rb

```diff
@@ -0,0 +1,26 @@
+class FriendshipsController < ApplicationController
```

Comment:
**[REQUIRED]**
* User can reject a friendship request ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/final_project/milestone5/accept_reject_friendship.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/49736080/)).
Let's implement `destroy` action so that a user can be able to cancel a friend request and update the views accordingly.